src/main/java/ru/itis/dto/ItemDto.java

```diff
@@ -1,22 +1,13 @@
 package ru.itis.dto;
 
-import lombok.Builder;
 import lombok.Data;
-import ru.itis.models.Item;
+import ru.itis.models.WishList;
 
 @Data
-@Builder
 public class ItemDto {
```

Comment:
The whole idea of DTO is a lame. So mappers are just a crutch

Reply:
sometimes it's a good solution